logging/channels_handlers.rst

```diff
@@ -178,7 +178,7 @@ How to Autowire Logger Channels
 Starting from `MonologBundle`_ 3.5 you can autowire different Monolog channels
 by type-hinting your service arguments with the following syntax:
 ``Psr\Log\LoggerInterface $<channel>Logger``. The ``<channel>`` must have been
-:ref:`predefined in your Monolog configuration <monolog-channels-config>`.
+:ref:`predefined in your Monolog configuration <monolog-channels-config>` and be in camelCase format.
```

Comment:
what about defining, in the exemple above, a `foo_bar` channel resulting in a `$fooBarLogger` instead?

Reply:
done